.circleci/config.yml

```diff
@@ -33,6 +38,14 @@ jobs:
       - run:
           name: Smoke test Echo API
           command: |
-            dockerize -wait tcp://0.0.0.0:3000 -timeout 1m
+            dockerize -wait tcp://localhost:16686 -timeout 1m # wait for jaeger
```

Comment:
dockerize looks really useful :)

Reply:
@PhilipGough  -> this will help us simplifying logic for flapping tests! :) 

https://github.com/jwilder/dockerize

example: 

`dockerize -wait tcp://db:5432 -wait http://web:80 -wait file:///tmp/generated-file`